gitlab.go

```diff
@@ -52,6 +52,22 @@ const (
 	OwnerPermission      AccessLevel = 50
 )
 
+// NotificationLevel represents a notification level within Gitlab.
+//
+// GitLab API docs: http://doc.gitlab.com/ce/...?
+type NotificationLevel int
+
+// List of available notification levels
+//
+// GitLab API docs: http://doc.gitlab.com/ce/...?
+const (
+	DisabledNotifications NotificationLevel = 0
+	ParticipatingNotifications NotificationLevel = 1
+	WatchNotifications NotificationLevel = 2
+	GlobalNotifications NotificationLevel = 3
+	MentionNotifications NotificationLevel = 4
+)
```

Comment:
Check... Well in this case I think it would be better/cleaner to use the `iota` solution instead. I looks like they are also using these values purely for enumeration, so I suggest we do the same.

The reasoning that it would be hard (counting the lines) to get the value is not applicable as that's the whole reason these constants exists right? So you don't have to memorise the numbers, but can use the written labels :wink: